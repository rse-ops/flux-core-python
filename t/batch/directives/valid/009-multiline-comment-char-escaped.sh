#!/bin/sh
# flux: -N4 --exclusive
# flux: --job-name=foo --setattr=user.conf="""
# flux: [config]
# flux:   item = "foo"  # an inline comment
# flux: # another comment
# flux: [tab2]
# flux:   b = 'bar'
# flux: """
foo
