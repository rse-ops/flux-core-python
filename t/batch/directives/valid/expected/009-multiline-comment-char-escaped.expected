SETARGS(['-N4', '--exclusive'])
SETARGS(['--job-name=foo', '--setattr=user.conf=[config]\n  item = "foo"  # an inline comment\n# another comment\n[tab2]\n  b = \'bar\'\n'])
